blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,158 @@
+title: "Meet bloop, a fast tool to compile and test your project"
+---
+
+Martin Duhem and I are happy to announce `bloop`, a tool that Scala developers
+can use to compile and test their projects faster without being tied to their
+stock build tool.
+
+## What is bloop?
+
+Bloop is a command-line tool for fast edit/compile/test workflows. Its primary goal is to compile
+and test your project as fast as possible, offering a snappy developer experience.
+
+## Why `bloop`?
+
+Build tools are complex applications that need to cover a wide range of use cases, from building and
+testing your project to deploying it, integrating with external tools, resolving dependencies and
+any build-related tasks.
+
+Such a vast array of requirements make build tools like `sbt`, `Maven` and `Gradle` create their own
+configuration models and DSLs to be customizable, whether that is programmatically or declaratively.
+It is easy that in the process of creating a build tool, performance is sacrificed by ease of
+customization, readability or maintenance.
+
+On the other hand, there is also the problem that build tools move slowly and
+have a hard time upgrading to new changes upstream (for example, in compiler
+APIs). As a result, `Gradle`, `Maven`, `Leiningen`, and `CBT` have not yet been
+migrated to Zinc 1.0, [which offers faster incremental compilations than its
+predecessor](https://www.scala-lang.org/blog/2017/11/03/zinc-blog-1.0.html). As
+a result, users of these tools are not benefiting from the latest state of the
+art.
+
+`bloop` aims to fix both problems. It only supports the most common build tasks
+(`compile`, `test` and `console`), and allows other build tools to integrate
+with it. You can think of `bloop` as a powered CLI for Zinc, rather than as
+a brand new build tool -- `bloop` does not aim to replace your stock build
+tool, rather complement it.
+
+In practice, this means that when users are only going to compile and test their project, they can
+use `bloop` instead of a full-blown resource-hungry build tool; and that generic build tools can
+delegate to `bloop` to compile and test users' projects instead of creating their own Scala and Java
+integrations, which are hard to maintain.
+
+## Numbers on compilation performance
+
+We created bloop because we observed that Scala compile times slow down when
+within sbt when compared to isolated benchmarks. This observation was at the
+beginning just an intuition, but when we sat down to measure a prototype we did
+see a significant difference.
+
+<blockquote class="twitter-tweet" data-lang="en-gb"><p lang="en" dir="ltr"> Scala Tooling has got to
+a point where people say "compiler performance" and what they really mean is "sbt performance".
+</p>&mdash; Jorge (@jvican) <a href="https://twitter.com/jvican/status/928601470232129536">9th November 2017</a></blockquote><script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+We'd like to share some of our numbers with you.
```

Comment:
Explain what you benchmark first! Before showing the results.

Reply:
You're also not explaining what you're doing. Be explicit. Say that you took these repos and you benchmarked fresh builds of the following projects. You can also say why you chose these projects.